bin/theme-setup.js

```diff
@@ -131,21 +133,157 @@ const promptThemeDataShort = ({devUrl, noConfirm}) => {
     }
 
     confirmed = summary([
+      { label: `${emoji.get('earth_africa')} Dev url`, variable: themeData.url },
+    ], noConfirm);
+  } while (confirmed !== 'y');
+
+  return themeData;
+};
+
+/**
+ * Prompts the user for all theme data (doesn't assume almost anything)
+ */
+const promptThemeData = ({ themeName, devUrl, noConfirm }) => {
+  let confirmed = 'n';
+  const themeData = {};
+
+  // -----------------------------
+  //  Prompt for project info
+  // -----------------------------
+
+  do {
+    if (!themeName) {
```

Comment:
```js
themeData.name = themeName;

if (!themeName) {
  themeData.name = promptFor({...}).trim();
}
```

Should work like the code in the `if/else` conditional, no?

Reply:
Yup, good catch, added that in. 👍